app/jobs/tsv_updater.rb

```diff
@@ -0,0 +1,35 @@
+require 'tempfile'
+require 'open-uri'
+
+class TsvUpdater < Updater
+  attr_reader :tempfile
+  attr_reader :tempfile_name
+  attr_reader :latest_url
+  attr_reader :importer
```

Comment:
Same as above; if you just want to enforce that implementing classes define `importer` you could get rid of this and do 

```
def importer
   raise StandardError.new('Must define #importer in subclass')
end
```

or similar. Same with the other elements of this interface (tempfile name, latest url, etc).